android/src/main/java/com/reactnativemsal/RNMSALModule.java

```diff
@@ -76,7 +78,7 @@ public void createPublicClientApplication(ReadableMap params, Promise promise) {
             // We first need to create the JSON model using the passed in parameters
 
             JSONObject msalConfigJsonObj = params.hasKey("androidConfigOptions")
-                    ? ReadableMapUtils.toJsonObject(params.getMap("androidConfigOptions"))
+                    ? ReadableMapUtils.toJsonObject(Objects.requireNonNull(params.getMap("androidConfigOptions")))
```

Comment:
Yeah maybe optimistic because when I look through it , here for example the `toJsonObject` method requires non nullable arguments and the value given is nullable (the map entries)- Android studio had listed that as a possible problem. There wasn't a handler for that and maybe this isn't the best solution but at least this guard ensures a check and maybe a faster invocation of the throw with `requireNonNull`